openshift/cronJob-template.yaml

```diff
@@ -82,10 +82,26 @@ objects:
                         secretKeyRef:
                           name: thoth
                           key: sentry-dsn
+                    - name: GRAPH_SERVICE_HOST
+                      valueFrom:
+                        configMapKeyRef:
+                          key: graph-service-host
+                          name: thoth
+                    - name: GRAPH_TLS_PATH
+                      value: "/var/run/secrets/dgraph"
+                  volumeMounts:
+                    - mountPath: /var/run/secrets/dgraph
+                      name: dgraph-tls-secrets
+                      readOnly: true
                   resources:
                     requests:
                       memory: "64Mi"
                       cpu: "100m"
                     limits:
                       memory: "128Mi"
                       cpu: "250m"
+                  volumes:
```

Comment:
It looks like volumes should go one level up.

Reply:
updated